contracts/bridge/BridgeTransferKLAY.sol

```diff
@@ -20,6 +20,43 @@ import "./BridgeTransferCommon.sol";
 
 
 contract BridgeTransferKLAY is BridgeTransfer {
+    bool public isLockedKLAY;
+
+    event KLAYLocked();
+    event KLAYUnlocked();
+
+    modifier lockedKLAY {
```

Comment:
```suggestion
    modifier onlyLockedKLAY {
```

Reply:
I think KLAY is one. So I think I don't need to write `only` for this.
What do you think? @jeonghonah

Reply:
It's okay :)